smart-contracts/rinkeby.json

```diff
@@ -0,0 +1 @@
{"balance":"0x1"},"00000000000000000000000000000000000000d3":{"balance":"0x1"},"00000000000000000000000000000000000000d4":{"balance":"0x1"},"00000000000000000000000000000000000000d5":{"balance":"0x1"},"00000000000000000000000000000000000000d6":{"balance":"0x1"},"00000000000000000000000000000000000000d7":{"balance":"0x1"},"00000000000000000000000000000000000000d8":{"balance":"0x1"},"00000000000000000000000000000000000000d9":{"balance":"0x1"},"00000000000000000000000000000000000000da":{"balance":"0x1"},"00000000000000000000000000000000000000db":{"balance":"0x1"},"00000000000000000000000000000000000000dc":{"balance":"0x1"},"00000000000000000000000000000000000000dd":{"balance":"0x1"},"00000000000000000000000000000000000000de":{"balance":"0x1"},"00000000000000000000000000000000000000df":{"balance":"0x1"},"00000000000000000000000000000000000000e0":{"balance":"0x1"},"00000000000000000000000000000000000000e1":{"balance":"0x1"},"00000000000000000000000000000000000000e2":{"balance":"0x1"},"00000000000000000000000000000000000000e3":{"balance":"0x1"},"00000000000000000000000000000000000000e4":{"balance":"0x1"},"00000000000000000000000000000000000000e5":{"balance":"0x1"},"00000000000000000000000000000000000000e6":{"balance":"0x1"},"00000000000000000000000000000000000000e7":{"balance":"0x1"},"00000000000000000000000000000000000000e8":{"balance":"0x1"},"00000000000000000000000000000000000000e9":{"balance":"0x1"},"00000000000000000000000000000000000000ea":{"balance":"0x1"},"00000000000000000000000000000000000000eb":{"balance":"0x1"},"00000000000000000000000000000000000000ec":{"balance":"0x1"},"00000000000000000000000000000000000000ed":{"balance":"0x1"},"00000000000000000000000000000000000000ee":{"balance":"0x1"},"00000000000000000000000000000000000000ef":{"balance":"0x1"},"00000000000000000000000000000000000000f0":{"balance":"0x1"},"00000000000000000000000000000000000000f1":{"balance":"0x1"},"00000000000000000000000000000000000000f2":{"balance":"0x1"},"00000000000000000000000000000000000000f3":{"balance":"0x1"},"00000000000000000000000000000000000000f4":{"balance":"0x1"},"00000000000000000000000000000000000000f5":{"balance":"0x1"},"00000000000000000000000000000000000000f6":{"balance":"0x1"},"00000000000000000000000000000000000000f7":{"balance":"0x1"},"00000000000000000000000000000000000000f8":{"balance":"0x1"},"00000000000000000000000000000000000000f9":{"balance":"0x1"},"00000000000000000000000000000000000000fa":{"balance":"0x1"},"00000000000000000000000000000000000000fb":{"balance":"0x1"},"00000000000000000000000000000000000000fc":{"balance":"0x1"},"00000000000000000000000000000000000000fd":{"balance":"0x1"},"00000000000000000000000000000000000000fe":{"balance":"0x1"},"00000000000000000000000000000000000000ff":{"balance":"0x1"},"31b98d14007bdee637298086988a0bbd31184523":{"balance":"0x200000000000000000000000000000000000000000000000000000000000000"}},"number":"0x0","gasUsed":"0x0","parentHash":"0x0000000000000000000000000000000000000000000000000000000000000000"}
```

Comment:
@julien I'm not sure what this file is? Other than that everything looks good.

Reply:
Ho, this is a seed `rinkeby.json` ;) I thought I'd add it here because if we need one day and can't get it from an official source it'll be handy to have it here!